index.js

```diff
@@ -57,12 +57,16 @@ function resolveUrlLoader(content, sourceMap) {
   loader.cacheable();
 
   // incoming source-map
-  var sourceMapConsumer, contentWithMap;
+  var sourceMapConsumer, contentWithMap, sourceRoot;
   if (sourceMap) {
 
     // sass-loader outputs source-map sources relative to output directory so start our search there
     try {
       relativeToAbsolute(sourceMap.sources, outputPath, resolvedRoot);
+      // There are now absolute paths in the source map so we don't need it anymore
+      // However, later when we go back to relative paths, we need to add it again
+      sourceRoot = sourceMap.sourceRoot;
+      sourceMap.sourceRoot = null;
     } catch (exception) {
```

Comment:
I agree, but I am not in a position to author right now.

Do you propose an integration test with webpack API?

So far I have a couple of open-source test projects that people have submitted me when they encounter a bug. At minimum I check against them to make sure I don't burn them twice. I will note them here later.